spec/definitions/Export.yaml

```diff
@@ -0,0 +1,69 @@
+type: object
+required:
+  - format
+properties:
+  id:
+    description: The export identifier string
+    readOnly: true
+    allOf:
+      - $ref: "#/definitions/ResourceId"
+  status:
+    description: Status of export request
+    readOnly: true
+    type: string
+    enum:
+      - "pending"
+      - "completed"
+      - "error"
+  Name:
+    description: Export's name
+    type: string
+  Resource:
+    description: Export's resource
```

Comment:
Would a better description be something like:

The type of resource being exported (eg., transaction).